### PR TITLE
Ultra Extended L1b Lookup tables

### PR DESCRIPTION
**Pull request by laspsandoval**
# Change Summary

## Overview
Breaking up PR #686. This is just a way to use the lookup tables. 

## New Files
lookup_utils.py
- Contains tools for lookup tables for l1b.

/lookup_tables (7 files)
- These were provided by the Ultra team.

## Testing

test_lookup_utils.py
- Tests lookup_utils.py

### CODE REVIEW

**Comment by laspsandoval** [comment]
@greglucas in the spirit of moving forward quickly could you approve if nothing else stands out?

File: imap_processing/tests/ultra/unit/test_lookup_utils.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+from imap_processing import imap_module_directory
+from imap_processing.ultra.l1b.lookup_utils import (
+    get_back_position,
+    get_energy_norm,
+    get_image_params,
+    get_norm,
+    get_y_adjust,
+)
+
+base_path = imap_module_directory / "ultra" / "lookup_tables"
```
Comment (tmplummer) [inline imap_processing/tests/ultra/unit/test_lookup_utils.py]: Global constants should be all uppercase.
```suggestion
BASE_PATH = imap_module_directory / "ultra" / "lookup_tables"
```

File: imap_processing/ultra/l1b/lookup_utils.py [hunk 2]
```diff
@@ -0,0 +1,160 @@
+"""Contains tools for lookup tables for l1b."""
+
+import numpy as np
+import pandas as pd
+
+from imap_processing import imap_module_directory
+
+base_path = imap_module_directory / "ultra" / "lookup_tables"
```
Comment (tmplummer) [inline imap_processing/ultra/l1b/lookup_utils.py]: Global constants should be all uppercase. Optionally starting with "_" like you did on the others below indicates it is a "private" global only to be used this module.
```suggestion
BASE_PATH = imap_module_directory / "ultra" / "lookup_tables"
```

**Review by tmplummer — APPROVED** [review]
This looks good to me. Two naming nit picks noted.

**Pull request merged**